pysonde/_helpers.py

```diff
@@ -236,17 +236,29 @@ def remove_nontype_keys(dict, allowed_type=type("str")):
     return {k: v for (k, v) in dict.items() if isinstance(v, allowed_type)}
 
 
+
 def remove_missing_cfg(cfg):
+
     """
     Remove config keys that are missing
     """
     return_cfg = {}
     for k in cfg.keys():
-        if OmegaConf.is_missing(cfg, k):
-            logging.warning(f"key {k} is missing and skipped")
-            pass
-        else:
-            return_cfg[k] = cfg[k]
+        try:
+            if OmegaConf.is_missing(cfg, k):
+                logging.warning(f"key {k} is missing and skipped")
+            else:
+                return_cfg[k] = cfg[k]
+            #need to add a if loop that check is the RS_type is a cor, if yes it should use a cfg with level0_cor instead of level0_mwx.yml
+            #to have the correct global attributes
+        except Exception as e:
```

Comment:
This raises actually an InterpolationKeyError which I now use in PR #26

Reply:
Nice work-around with setting the value to 'no info'. I adjusted it slightly in integrated it in PR #26